src/WebServer/Rest/RestControllerMethodInfo.cs

```diff
@@ -20,9 +20,9 @@ internal enum TypeWrapper
         private readonly IEnumerable<Type> _validParameterTypes;
 
         private readonly UriParser _uriParser;
-        private readonly ParsedUri _matchUri;
         private readonly IEnumerable<ParameterValueGetter> _parameterGetters;
 
+        internal ParsedUri MatchUri { get; }

```

Comment:
Don't get why this should be internal, isn't used outside class and not in a test method. What am I missing?